strawberryfields/apps/__init__.py

```diff
@@ -0,0 +1 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
```

Comment:
Note: since this is now part of Strawberry Fields, it makes sense to specify the license at the top of each file in `apps/`. You can just copy the Apache 2.0 license from other Python files. If we wish to change the license on the `apps/` subfolder at some point, the Apache license is permissive and will allow this.